TrainingCenter/models/Course.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace TrainingCenter.models;
+
+public class Course
+{
+    private string title;
+    private List<Lesson> lessons;
+    private List<Teacher> teachers;
+
+    public Course(string title)
+    {
+        this.title = title;
+        this.lessons = new List<Lesson>();
+        this.teachers = new List<Teacher>();
+    }
+
+    public Course(string title, List<Lesson> lessons)
+    {
+        this.title = title;
+        this.lessons = lessons;
+    }
+
+    public void AddLesson(Lesson lesson)
+    {
+        lessons.Add(lesson);
```

Comment:
Probably here it would be good to check if there is not already such lesson in the list